test/mux.sh

```diff
@@ -16,10 +16,14 @@ else
     exit 0
 fi
 
-FLUENTD_WAIT_TIME=$(( 2 * minute ))
+FLUENTD_WAIT_TIME=${FLUENTD_WAIT_TIME:-$(( 2 * minute ))}
 
 os::test::junit::declare_suite_start "test/mux"
 
+ARTIFACT_DIR=${ARTIFACT_DIR:-${TMPDIR:-/tmp}/origin-aggregated-logging}
+
+os::log::debug ARTIFACT_DIR is $ARTIFACT_DIR
```

Comment:
remove these - just use the default ARTIFACT_DIR set by init.sh

Reply:
Actually, just rebase on top of https://github.com/openshift/origin-aggregated-logging/pull/754 now that it has merged

Reply:
Yep, I revived it... :p  Sorry, removing it again.